### ISSUE TITLE
BUG: why no ROIs are labeled after data processing

### ISSUE THREAD

**Issue by yduxx** [MouseLand/suite2p#1054]
### Describe the issue:

I just installed the suite2p. the installment seems going well.

after I finish the image registration, Ca extraction and all the data processing procedures, no ROIs are labeled.  but there are identified cells and non-cells indicated above the image zone.
all seem going well except for the labeled ROI.

when I click on the ROIs on[space bar]， nothing happened in the image zone.

### Reproduce the code example:

```python
in the anaconda prompt, it reported
 "QGraphicsScene::removeItem: item 0x21a4c41a370's scene (0x0) is different from this scene (0x21a4ab4f4e0)
QGraphicsScene::removeItem: item 0x21a4c6dbba0's scene (0x0) is different from this scene (0x21a4ab4f4e0)
", every time when I click on the "ROIs on[space bar]".
```

### Version information:

suite2p v0.14.2

**Comment by Andrianarivelo** [comment]
I have the same issue on windows 10 even after following this : 
```
conda create --name suite2p python=3.9
conda activate suite2p
pip install PyQt-builder
python -m pip install suite2p[gui]
python -m pip install suite2p[io]
python -m suite2p
```

**Comment by landoskape** [comment]
There is a solution to this issue in PR #1029.

**Comment by Andrianarivelo** [comment]
Hi thanks for the response, I don't get the fix...

**Comment by landoskape** [comment]
@yduxx see the discussion in #1028 for a solution

**Comment by chriski777** [comment]
Hi @Andrianarivelo, as mentioned by @landoskape, the fix in #1029 has been merged to the main branch. Please pull the latest changes from the main branch and try after rebuilding suite2p.
